### ISSUE TITLE
Can be tag clouds hidden for non loged users?

### ISSUE THREAD

**Issue by 321jaber** [RobertHeim/phpbb-ext-topictags#96]
Is there some parameter to make cloud tags hidden for non loged users?
![Req1](https://user-images.githubusercontent.com/81232806/112139572-36e0e200-8bd3-11eb-9c26-fe571ff65b4e.PNG)

phpbb 3.3.3 php 7.4

**Comment by SMcCandlish** [comment]
Yes, this is pretty important.  Some boards are private for good reason, and it's not desirable to have to choose between a fully public tag cloud and no tag cloud at all on the front page.  Geniuses of phpBB customization can probably figure out how to hack in a permissions check here, but it would be better if the extension just had an option for it.
